src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -3,13 +3,17 @@
     full-screen
     class="confirm-transaction-sign"
     data-cy="popup-aex2"
-    absolute-footer
   >
-    <TransactionOverview :tx="completeTransaction" />
+    <TransactionOverview
+      class="transaction-overview"
```

Comment:
But transaction overview component already have a `transaction-overview` class.

Reply:
It doesn't matter. If we are styling some class name within the <style> block the class should exist in the scope of the <template> block. So we should avoid styling stuff out of the current context. This is related to the "encapsulation" rules.

When I joined the team I've seen a lot of places where the parent component was trying to apply some styling on class names that should exists in the child scope, but someone changed the name or the structure of the child component so at the end it was messed up.

In general parent components should be blind for the child component's internals.

Reply:
- Now all code is consistent
- With that approach we would still need to rename classes names in case we would rename the component to at least match the logic
- In that case we would need to rewrite the whole project in that to be consistent

Would this approach will bring us more value than it would took effort?

Reply:
I removed my change from the PR on rounded corners. There is an open conversation on how we should behave. But I think it should not block the task.